docs/data-sources/container.md

```diff
@@ -60,27 +65,27 @@ In addition to all arguments above, the following attributes are exported:
 
 - `environment_variables` - The [environment](https://www.scaleway.com/en/docs/compute/containers/concepts/#environment-variables) variables of the container.
 
-- `min_scale` - The minimum of running container instances continuously. Defaults to 0.
+- `min_scale` - The minimum number of container instances running continuously. Defaults to 0.
 
-- `max_scale` - The maximum of number of instances this container can scale to. Default to 20.
+- `max_scale` - The maximum number of instances the container can scale to. Defaults to 20.
 
-- `memory_limit` - The memory computing resources in MB to allocate to each container. Defaults to 128.
+- `memory_limit` - The memory resources in MB to allocate to each container. Defaults to 256.
```

Comment:
Where do you get this information from? It needs to be consistant with the documentation inside the schema.

https://github.com/scaleway/terraform-provider-scaleway/blob/master/internal/services/container/container.go#L95